pkg-scripts/vircadia-assignment-client.service

```diff
@@ -12,7 +12,7 @@ User=vircadia
 Group=vircadia
 #LimitCORE=infinity
 #ExecStart=/opt/vircadia/assignment-client -n 6
-ExecStart=/opt/vircadia/assignment-client --min 6 --max 20
+ExecStart=/opt/vircadia/assignment-client --min 6 --max 20 --disable-domain-port-auto-discovery
```

Comment:
Please remove this change; the service without the @ is supposed to operate without the multi-domain workarounds.